src/app/code/community/Ambimax/PriceImport/Model/ErpImport.php

```diff
@@ -109,9 +108,9 @@ public function loadCsvData($fileLocation)
      * @param $additional boolean
      * @return string
      */
-    public function getLocalFilePath()
+    public function getLocalFilePath($fileName)
     {
-        return Mage::getStoreConfig('ambimax_priceimport/erp_import_options/file_path');
+        return Mage::getStoreConfig('ambimax_priceimport/erp_import_options/file_path') . $fileName;
```

Comment:
What about a missing trailing `/` in `'ambimax_priceimport/erp_import_options/file_path'`?